config/config.yml

```diff
@@ -0,0 +1,14 @@
+ingest_data:
```

Comment:
Will the user need to change this file at all?

Reply:
The user won't have to, these can serve as the defaults, but they can be changed to adjust the behavior of some scripts (where to save the data, how to create the database, etc.).